apps/wing/init-templates/wing/http-api/main.w

```diff
@@ -0,0 +1,27 @@
+bring cloud;
```

Comment:
I think the template will likely also need to include a `package.json` file with the relevant config and when running `wing new`, it should execute `npm install`.

Reply:
Added an install step